packages/sheets/src/commands/commands/set-border-command.ts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+/* eslint-disable max-lines-per-function */
```

Comment:
Do not add this. Please break long functions into smaller ones to make the code readable.

Reply:
<img width="1530" alt="image" src="https://github.com/user-attachments/assets/04348863-99ae-42c8-822f-92f2713c31fa">
break into multiple function